includes/syslog.php

```diff
@@ -57,9 +57,15 @@ function process_syslog($entry, $update)
     }
 
     $entry['host'] = preg_replace("/^::ffff:/", "", $entry['host']);
-    if ($new_host = Config::get("syslog_xlate.{$entry['host']}")) {
-        $entry['host'] = $new_host;
+    $axlate = Config::get('syslog_xlate');
+    if(true == is_array($axlate)){
```

Comment:
This entire thing could be replaced with something like
```Php
if (is_array($axlate) && isset($axlate[$entry['host']]) {
  $entry['host'] = $axlate[$entry['host']];
}
```
If im not missing anything.